### DISCUSSION TITLE
Output LEvel

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#842] [Bug Reports]
Looks from the latest update the ouput level is lower than before ?

Does anybody heard that ?

Thanks

**Comment by cYCL157** [comment]
I had thought the same with a recent version -- tried everything I could think of -- I ended up disconnecting cables and reconnecting and then things seemed to work.  Not sure how that would have been the case with an update, but.........

Here was the issue I reported, rern has some suggestions that might be worth trying:  https://github.com/rern/rAudio-1/discussions/809

**Comment by Rasb74** [comment]
I've made an easy test : Reloading a brand new SD card, with a previous R-Audio image : Everything is ok with the previous image 
Then Really think the latest update affect the output level.

Rern :Any idea ?

Thanks

Pascal

**Comment by rern** [comment]
Try: `alsamixer`  > `F6` to select sound card > `arrow up` / `arrow down`

**Reply by Rasb74** [reply]
Tried to manage on alsamixer
volume. to reacting, 
the filter settings is not reacting too. (In the future it will be intesresçtng to have access in the Gui of the filter settings of the day chipset
